korman/properties/prop_material.py

```diff
@@ -0,0 +1,27 @@
+#    This file is part of Korman.
+#
+#    Korman is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU General Public License as published by
+#    the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    Korman is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License
+#    along with Korman.  If not, see <http://www.gnu.org/licenses/>.
+
+import bpy
+from bpy.props import *
+
+from .. import idprops
+
+class PlasmaMaterial(bpy.types.PropertyGroup):
+    bl_idname = "mat.plasma_mat"
```

Comment:
```suggestion
    bl_idname = "material.plasma_material"
```

Reply:
I was conflicted on that as I wasn't sure if using "material" would affect anything. I'll change that.